sw/device/lib/runtime/check.h

```diff
@@ -37,6 +37,6 @@
  * @param ... arguments to a LOG_* macro, which are evaluated if the check
  * fails.
  */
-#define CHECKZ(value, ...) CHECK((value) == 0, ...)
```

Comment:
Should be a separate commit.

Reply:
It's going to be a separate PR. I just haven't gotten around to it, because this is still a draft.